packages/crypto/src/__tests__/crypto.test.ts

```diff
@@ -8,12 +8,21 @@ import {
   PhraseSize,
   Rng,
   Salt,
-  ShieldedHDWallet,
+  ShieldedHDWallet
 } from "../crypto/crypto";
 
+import {
+  readStringPointer,
+  readVecStringPointer,
+  readVecU8Pointer
+} from "../utils";
+
+// __wasm is not exported in crypto.d.ts so need to use require instead of import
+const memory = (require("../crypto/crypto")).__wasm.memory;
+
 const KEY_LENGTH = 32;
 const SEED_LENGTH = 64;
-const SHIELDED_KEY_LENGTH = 96;
+const SHIELDED_CHILD_KEY_LENGTH = 169;
```

Comment:
I don't know where this number comes from and if it is stable to use, but I just took it from the actual output so the tests pass.

Reply:
Had no idea, but it seems to be correct :)
Summing all the bytes in:
```
pub struct ExtendedFullViewingKey {
    depth: u8,
    parent_fvk_tag: FvkTag,
    child_index: ChildIndex,
    chain_code: ChainCode,
    pub fvk: FullViewingKey,
    dk: DiversifierKey,
}
```
Returns 169